internal/supported_types/aws_supported_types.go

```diff
@@ -0,0 +1,18 @@
+package supported_types
```

Comment:
If anyone has a better name, please submit. Idiomatic Go is to have short packages (e.g. fmt instead of format). Maybe "instance" or "inst"?